lib/protobuf/extension.ex

```diff
@@ -131,12 +129,18 @@ defmodule Protobuf.Extension do
         fnum = ext.field_props.fnum
         fnum_key = {Protobuf.Extension, ext.extendee, fnum}
 
-        if GlobalStore.get(fnum_key, nil) do
+        if :persistent_term.get(fnum_key, nil) do
           raise "Extension #{inspect(ext.extendee)}##{fnum} already exists"
         end
 
-        GlobalStore.put(fnum_key, mod)
+        :persistent_term.put(fnum_key, mod)
       end)
     end)
   end
+
+  def unload_extensions do
```

Comment:
**chore (blocking)**: If this is public, then we need to add a spec and docs. If it's only meant for testing, then let's add `@doc false` and a commend saying `# Made public for testing`.

Reply:
Good catch! I believe visibility is public here not because of testing but actually because we call these functions (both `cal_extensions/1` and `unload_extensions/0`) from `Protobuf.Application`. Maybe we should rename them to `__cal_extensions__/1` and `__unload_extensions/0` too, to make it clearer it is part of a private API?

Reply:
Yes, let's rename to `__...__`, make them `@doc false` 👍